engine/graphics/src/vulkan/graphics_vulkan_private.h

```diff
@@ -100,6 +92,7 @@ namespace dmGraphics
         VkImageUsageFlags m_UsageFlags;
         DeviceBuffer      m_DeviceBuffer;
         int32_atomic_t    m_DataState; // data state per mip-map (mipX = bitX). 0=ok, 1=pending
+        HOpaqueHandle     m_PendingUpload;
```

Comment:
This is the upload marker for the GPU upload. It's a slightly different setup than our normal async uploads, this system is mostly for avoiding GPU stalls on the main thread.